platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -51,6 +51,7 @@ NS_INLINE mbgl::EdgeInsets MGLEdgeInsetsFromNSEdgeInsets(NSEdgeInsets insets) {
     @param latitude The latitude of the point at the center of the viewport.
     @param size The size of the viewport.
     @return An altitude measured in meters. */
+MGL_EXTERN
```

Comment:
Ah, OK, so we need this on anything with Private visibility but not Project visibility (using Xcode terminology here).